habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -162,7 +162,10 @@ def calculate_turn_pose(self, target_position: mn.Vector3):
         self.calculate_walk_pose(target_position, distance_multiplier=0)
 
     def calculate_walk_pose(
-        self, target_position: mn.Vector3, distance_multiplier=1.0
+        self,
+        target_position: mn.Vector3,
+        distance_multiplier=1.0,
+        target_dir=None,
```

Comment:
Can you update the function comment explaining what `target_dir` does? It seems nearly identical to `target_position` in terms of how its used inside this function.

Reply:
Sounds good! See my comment above. I will update the function with a comment too